app/locales/en.js

```diff
@@ -0,0 +1,40 @@
+export default {
```

Comment:
Perhaps not at this stage, but is it worth moving language files to their own repo? https://github.com/zooniverse/translate-pfe